release_notes/47-release-notes.adoc

```diff
@@ -478,4 +480,30 @@ This release provides fixes for the following security issues:
 * zlib: Out-of-bounds pointer arithmetic in `inftrees.c` link:https://access.redhat.com/security/cve/CVE-2016-9840[(CVE-2016-9840)]
 * krb5: Kerberos RC4-HMAC-MD5 checksum vulnerability link:https://access.redhat.com/security/cve/CVE-2025-3576[(CVE-2025-3576)]
 
+[id="about-release-475_{context}"]
+== About release version 4.7.5
+
+*Release date*: 15 July 2025
+
+This release of {product-title-short} 4.7 provides the following bug fixes:
+
+//ROX-29320
+* Before this update, incorrect interpretation of Red Hat Enterprise Linux (RHEL) 10 Common Platform Enumeration (CPE) strings caused Scanner V4 to fail distribution checks on RHEL 10 systems. With this update, an updated RHEL CPE major version pattern resolves the issue, and Scanner V4 can now correctly support RHEL 10.
+
+//ROX-29755
+* Before this update, the failure of Sensor to call stream.Recv() caused gRPC flow control to block image reprocessing every 4 hours. With this update, the reprocessing loop includes a timeout for sending messages to Sensors, which resolves the issue and and resumes the image reprocessing as expected.
+
+//ROX-28665
+* Before this update, missing network policy configuration for port `9090` on long-running clusters from earlier releases prevented Elastic search from indexing those clusters. With this update, Elastic search integration for long-running cluster metrics resolves the issue, and you can now access search results from these clusters.
```

Comment:
```suggestion
* Before this update, missing network policy configuration for port 9090 on long-running clusters from earlier releases prevented Elasticsearch from indexing those clusters. With this update, Elasticsearch integration for long-running cluster metrics resolves the issue, and you can now access search results from these clusters.
```

Reply:
> Afaik this is only internal and does not affect any customers, as they would never deploy a long-running cluster and thus never extract metrics into Elastic Search. I would suggest to skip this paragraph. Pls check me @JoukoVirtanen

Customers do not run long-running clusters, so there is no reason for this to be in the user facing documentation.

Reply:
I am confused, the conversation is resolved, but the text is still there. We suggested to remove that paragraph totally. Did you forget to push maybe?